docs/userguide/labels-custom-metadata.md

```diff
@@ -11,34 +11,34 @@ parent = "mn_use_docker"
 # Apply custom metadata
 
 You can apply metadata to your images, containers, or daemons via
-labels. Metadata can serve a wide range of uses. Use labels to add notes or
-licensing information to an image or to identify a host.
+labels. Labels can serve a wide range of uses cases, they can be use to specify
```

Comment:
s/uses cases/uses/  (it's not a use *case* here)
s/can be use/can be used/

Alternatively;

```
Labels serve a wide range of uses, such as adding notes or licensing information
to an image, or to identify a host.
```